Testscripts/Linux/XDP-Action.sh

```diff
@@ -0,0 +1,301 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# This script verifies XDP Action - TX or DROP
+# For DROP Action verification script will compare number of ping packets
+#		lossed before XDP DROP action and after XDP DROP action
+# For TX Action verification script will start ping on server and measure the packets captured by tcpdump
+#		Expected result: 0 packets should be captured by tcpdump after TX action as this action
+#		will return all ping packets to sender at hardware level
+
+# This function creates patch file for xdpdump application
+# depending upon which action we are validating
+function Get-XDPDump-Patch () {
```

Comment:
Is the patch available in linux-next or another other upstream branch? I am not sure if this is right approach. Generating a patch in the run time? If failed, ?

Reply:
These patches are for xdpdump application. Out of the box this application ships with XDP action "PASS". For validating scenarios of DROP and TX actions @haiyangz  has created patches for xdpdump to modify ICMP packets received at hardware level.